scipy/sparse/linalg/_norm.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 from scipy.sparse import issparse
 
-from numpy.core import Inf, sqrt, abs
+from numpy import Inf, sqrt, abs
```

Comment:
This is the only actual change-everything else was my delete-whitespace-upon-save hook. That part can be removed if desired.